src/components/CallScriptsCard.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div>
+    <div class="scriptCordion">
+      <!--- Inbound Call Button --->
+      <base-button
+        variant="text"
+        class="w-full py-3 font-bold text-crisiscleanup-dark-400"
+        :action="() => (toggleInbound = !toggleInbound)"
+        :ccu-icon="[toggleInbound ? 'up' : 'down']"
```

Comment:
think you meant `( )` here, cause your declaring an array here.

Reply:
> 
> 
> Also, if you could start adding screenshots / gifs of the components referenced so that @arroyoDev and @aarontitus can easily review them, that'd be awesome.
> 
> See #158 as an example

gotcha, will do